demos/03_license_plate_pipe/setup/README.md

```diff
@@ -0,0 +1,4 @@
+#Steps to setup3
+oc new-project pipelines-demo
+oc apply -f setup/workspace-pvc.yaml
+oc apply -f pipeline/licenseplate-pipeline.yaml 
```

Comment:
Perhaps you could write some words about the PVC etc. Just quickly describe how to execute the demo.